server.py

```diff
@@ -63,6 +64,7 @@ def write_file():
 def read_file():
     ip_address = request.remote_addr if request.args.get('ip') is None else request.args.get('ip')
     metadata = getattr(g, 'metadata', None)
+    delay_time = 0 if request.args.get('delay') is None else int(request.args.get('delay'))
```

Comment:
We should remember to update the wiki for the server with this and with other info as well (e.g. `/can_move_file`)